src/frontend/e2e/utils/schemas/sampleResponseDefaults.ts

```diff
@@ -1,11 +1,11 @@
 export type SampleResponseDefaults = {
   collection_date?: string;
   collection_location?: number;
-  czb_failed_genome_recovery?: boolean;
   gisaid_id?: string | null;
   gisaid_status?: string;
   id?: number;
-  lineage?: string;
+  lineages?: [Lineage];
```

Comment:
The way these types are written indicate that `lineages` is a one-item tuple, where the only item is a `Lineage`. Is that what you meant? If you meant a Lineage array, this should be changed to `Lineage[]`

Reply:
yeah because as of now there should only be one Lineage entry per sample. We were future proofing the backend by having the capability to have a list of associated lineages or qc_metrics in case we wanted to implement that in the future, but for right now there should only be one Lineage or qcMetric per sample